internal/api/photo_count_per_month.go

```diff
@@ -0,0 +1,31 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/photoprism/photoprism/internal/config"
+	"github.com/photoprism/photoprism/internal/query"
+	"github.com/photoprism/photoprism/pkg/txt"
+
+	"github.com/gin-gonic/gin"
+)
+
+// GET /api/v1/moments/time
+func GetPhotoCountPerMonth(router *gin.RouterGroup, conf *config.Config) {
```

Comment:
Can you also rename this to `GetMomentsTime()`? Looks good otherwise, probably need to add fixtures to properly test this with unit tests... we can do this later, so that you can focus on developing our mobile app :)

Reply:
sound good :+1: